### PR TITLE
enh(ui): Use centreon-ui for all top counters

### PR DESCRIPTION
**Pull request by Thebarda**
# Pull Request Template

## Description

This PR use centreon-ui's component inside centreon's top counters to match with centreon-bam's top counter. Also this PR does not change any functionality about these top counters.

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

You can test this PR by using Hosts and Services top counters. Visually, every top counter should be the same (same colors, same space between top counters, same space between icons, same number icon and same text style)

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by Thebarda** [comment]
Here is performance comparison between this branch and master.
Note : Tested on a PC with two VSCode, a terminal and a slack that is running. Also CPU frequence is not fixed so it could be a margin of error. Finally, I tested on Firefox.

This branch : 
- CPU (%) : 3.60
- MEM (Mo): 20

Master:
- CPU (%) : 3.35
- MEM (Mo) : 19.3

**Comment by bdauria** [comment]
blocked by https://github.com/centreon/centreon/tree/rename_%40centreon/react-components-to_%40centreon/ui (to be merged first)

File: www/front_src/src/components/hostMenu/index.js [hunk 1]
```diff
@@ -121,191 +127,114 @@ class HostMenu extends Component {
...
+                />
+              </Link>
+              <Link
+                to="/main.php?p=20202&o=h_unreachable&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  dotColored="gray"
+                  submenuTitle="Unreachable"
```
Comment (kduret) [inline www/front_src/src/components/hostMenu/index.js]: should be translated

File: www/front_src/src/components/hostMenu/index.js [hunk 2]
```diff
@@ -121,191 +127,114 @@ class HostMenu extends Component {
...
+                />
+              </Link>
+              <Link
+                to="/main.php?p=20202&o=h_down&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  dotColored="red"
+                  submenuTitle="Critical"
```
Comment (kduret) [inline www/front_src/src/components/hostMenu/index.js]: should be translated

File: www/front_src/src/components/hostMenu/index.js [hunk 3]
```diff
@@ -121,191 +127,114 @@ class HostMenu extends Component {
...
-                  styles['list-unstyled'],
-                )}
+            <SubmenuItems>
+              <Link
+                to="/main.php?p=20202&o=h&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  submenuTitle="All"
```
Comment (kduret) [inline www/front_src/src/components/hostMenu/index.js]: should be translated

File: www/front_src/src/components/hostMenu/index.js [hunk 4]
```diff
@@ -121,191 +127,114 @@ class HostMenu extends Component {
...
+                />
+              </Link>
+              <Link
+                to="/main.php?p=20202&o=h_up&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  dotColored="green"
+                  submenuTitle="Up"
```
Comment (kduret) [inline www/front_src/src/components/hostMenu/index.js]: should be translated

File: www/front_src/src/components/hostMenu/index.js [hunk 5]
```diff
@@ -121,191 +127,114 @@ class HostMenu extends Component {
...
-                        {numeral(data.pending).format()}
-                      </span>
-                    </div>
-                  </Link>
-                </li>
-              </ul>
-            </div>
+                <SubmenuItem
+                  dotColored="blue"
+                  submenuTitle="Pending"
```
Comment (kduret) [inline www/front_src/src/components/hostMenu/index.js]: should be translated

File: www/front_src/src/components/serviceStatusMenu/index.js [hunk 6]
```diff
@@ -126,242 +132,147 @@ class ServiceStatusMenu extends Component {
...
-                  styles['list-unstyled'],
-                )}
+            <SubmenuItems>
+              <Link
+                to="/main.php?p=20201&o=svc&statusFilter=&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  submenuTitle="All Services"
```
Comment (kduret) [inline www/front_src/src/components/serviceStatusMenu/index.js]: should be translated

File: www/front_src/src/components/serviceStatusMenu/index.js [hunk 7]
```diff
@@ -126,242 +132,147 @@ class ServiceStatusMenu extends Component {
...
+                />
+              </Link>
+              <Link
+                to="/main.php?p=20201&o=svc_unhandled&statusFilter=critical&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  dotColored="red"
+                  submenuTitle="Critical services"
```
Comment (kduret) [inline www/front_src/src/components/serviceStatusMenu/index.js]: should be translated

File: www/front_src/src/components/serviceStatusMenu/index.js [hunk 8]
```diff
@@ -126,242 +132,147 @@ class ServiceStatusMenu extends Component {
...
+                />
+              </Link>
+              <Link
+                to="/main.php?p=20201&o=svc_unhandled&statusFilter=warning&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  dotColored="orange"
+                  submenuTitle="Warning services"
```
Comment (kduret) [inline www/front_src/src/components/serviceStatusMenu/index.js]: should be translated

File: www/front_src/src/components/serviceStatusMenu/index.js [hunk 9]
```diff
@@ -126,242 +132,147 @@ class ServiceStatusMenu extends Component {
...
+                />
+              </Link>
+              <Link
+                to="/main.php?p=20201&o=svc_unhandled&statusFilter=unknown&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  dotColored="gray"
+                  submenuTitle="Unknown services"
```
Comment (kduret) [inline www/front_src/src/components/serviceStatusMenu/index.js]: should be translated

File: www/front_src/src/components/serviceStatusMenu/index.js [hunk 10]
```diff
@@ -126,242 +132,147 @@ class ServiceStatusMenu extends Component {
...
+                />
+              </Link>
+              <Link
+                to="/main.php?p=20201&o=svc&statusFilter=ok&search="
+                className={styles.link}
+                onClick={this.toggle}
+              >
+                <SubmenuItem
+                  dotColored="green"
+                  submenuTitle="Ok services"
```
Comment (kduret) [inline www/front_src/src/components/serviceStatusMenu/index.js]: should be translated

File: www/front_src/src/components/serviceStatusMenu/index.js [hunk 11]
```diff
@@ -126,242 +132,147 @@ class ServiceStatusMenu extends Component {
...
-                        {numeral(data.pending).format()}
-                      </span>
-                    </div>
-                  </Link>
-                </li>
-              </ul>
-            </div>
+                <SubmenuItem
+                  dotColored="blue"
+                  submenuTitle="Pending services"
```
Comment (kduret) [inline www/front_src/src/components/serviceStatusMenu/index.js]: should be translated

**Comment by kduret** [comment]
You should use I18n.t function instead of Translate component.
ike it's done here ;) https://github.com/centreon/centreon/blob/master/www/front_src/src/components/pollerMenu/index.js#L75

**Comment by cgagnaire** [comment]
> 
> 
> You should use I18n.t function instead of Translate component.
> ike it's done here ;) https://github.com/centreon/centreon/blob/master/www/front_src/src/components/pollerMenu/index.js#L75

Isn't it a good time to remove the extra "services" word that is useless here and does not match the hosts menu.

**Comment by Thebarda** [comment]
> > You should use I18n.t function instead of Translate component.
> > ike it's done here ;) https://github.com/centreon/centreon/blob/master/www/front_src/src/components/pollerMenu/index.js#L75
> 
> Isn't it a good time to remove the extra "services" word that is useless here and does not match the hosts menu (ref [f1efa55](https://github.com/centreon/centreon/commit/f1efa5572a7b38cc6162272cc7aa3768e5b361da)).

Should I remove translations ?

**Comment by Thebarda** [comment]
A lot of acceptance tests do not pass on the last build : https://jenkins.centreon.com/blue/organizations/jenkins/centreon-web/detail/PR-8218/4/pipeline
I don't know what happened on that build but I bet that is not due to my last changes.

**Pull request merged**